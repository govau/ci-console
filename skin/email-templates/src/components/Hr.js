import React from "react";
import styled from "styled-components";

const StyledHr = styled.hr`
  margin: 20px 0 40px 0;
  border-top: 0;
  border-color: #f0f0f0;
`;

export default () => <StyledHr />;
