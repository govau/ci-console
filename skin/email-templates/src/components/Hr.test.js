import React from "react";
import renderer from "react-test-renderer";

import Hr from "./Hr";

test("renders without crashing", () => {
  const tree = renderer.create(<Hr />);
  expect(tree).toMatchSnapshot();
});
