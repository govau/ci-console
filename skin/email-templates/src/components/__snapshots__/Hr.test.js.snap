// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.c0 {
  margin: 20px 0 40px 0;
  border-top: 0;
  border-color: #f0f0f0;
}

<hr
  className="c0"
/>
`;
